src/containers/Card.js

```diff
@@ -1,5 +1,5 @@
 import React, { PropTypes } from 'react';
-import { View, StyleSheet, Platform, Image } from 'react-native';
+import { View, StyleSheet, Platform, Image,Text as NativeText } from 'react-native';
```

Comment:
missing a space. Also won't have to worry about this int he future as we are adding prettier.